### DISCUSSION TITLE
登录成功 为什么不跳转呢

### DISCUSSION THREAD

**Discussion by YuXunSTC** [1Panel-dev/1Panel#5816] [Q&A]
### 联系方式

1

### 1Panel 版本

v1.10.11

### 问题描述

<img width="738" alt="Snipaste_2024-07-15_09-44-48" src="https://github.com/user-attachments/assets/ad51864b-768b-4a57-92fe-a7aa369aabb5">

### 重现步骤

1

### 期待的正确结果

1

### 相关日志输出

```shell
1
```

### 附加信息

1

**Comment by YuXunSTC** [comment]
chrome 浏览器

**Comment by YuXunSTC** [comment]
换了个 safari 可以跳转了

**Comment by YuXunSTC** [comment]
昨天可以跳转，今天数据库总是外网访问，于是重装了系统，然后就无法跳转了 ，chrome 浏览器 清除了缓存也不管用

**Comment by zhengkunwang223** [comment]
无痕模式试一下
